public/javascripts/labelingGuidePanelResize.js

```diff
@@ -0,0 +1,49 @@
+function checkWindowSize(){
```

Comment:
Could you add a comment describing what this function does? And does it actually just check the window size and report back? Or does it do stuff after it checks the window size? If it checks the size and then does stuff, maybe split that into two separate functions? Or if the main point of the function is doing "X", just rename the function to reflect that.